build.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+if [ "$(uname)" == "Darwin" ]; then
+    #Mac OS X platform
+	yes | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
>yes | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" [](start = 1, length = 104)

Can we have comment why we doing this?
I would assume this is LightGBM dependency, but I would prefer to make this statement here, instead of going through file history